daliuge-common/dlg/common/__init__.py

```diff
@@ -71,13 +71,18 @@ def b2s(b, enc='utf8'):
         return b.decode(enc)
     def u2s(u):
         return u
+    def s2b(s, enc='utf-8'):
+        return s.encode(enc)
```

Comment:
`s2b` is basically what `six.b` does (except six uses `latin1` when encoding). If possible could we use that instead? Unless of course there are non-latin1 characters in your strings, in which case this *is* necessary.

Reply:
I don't think there will be any non-latin1 characters. I'll remove common.s2b and use six.b instead.